src/main/java/mate/service/DriverServiceImpl.java

```diff
@@ -35,4 +35,11 @@ public Driver update(Driver driver) {
     public boolean delete(Long id) {
         return driverDao.delete(id);
     }
+
+    public Driver createInstance(String name, String licenseNumber) {
```

Comment:
Why do we need to add this method? The code looks like similar constructor that can be located in the `model` package.